packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -354,13 +356,9 @@ const helpers = {
                 });
               } else {
                 return pipe(
-                  findEmptyResponse(
-                    response,
-                    response.headers || [],
-                    get(contentWithExamples, 'mediaType', get(responseWithSchema, 'schema')) || ['*/*']
```

Comment:
`contentWithExamples` is always undefined in this branch, that's why I've removed it